### PR TITLE
Dokumentation Readme

### LINKED ISSUE #146: ReadMe schreiben
baseline anpassen weil wir die baseline of OmegaPrint haben

außerdem sollten wir noch was zur Benutzung schreiben

### PR DESCRIPTION
**Pull request by PatriciaSowa**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,10 +2,20 @@
...
 	load
 ```
+
+After installing OmegaPrint you will find a button in all your code holder. For example the browser. 
+Clicking on it OmegaPrint will show you a prettified version of your code.
+
+##Comments
+OmegaPrint does basic comment-formatting.
+There is a problem because comments are not in the CST:
+Therefore, we use the method OhmNode >> skippedSpacesNodes which gives us Nodes containing all spaces and comments, which are before the Node on which we call the method.
```
Comment (Paula-Kli) [inline README.md]: Comment-formatting is more complicated than code-formatting. This happens due to the fact, that comments and whitespace is not in the CST. Therefore...

**Comment by Paula-Kli** [comment]
Vielleicht könnte man noch was schreiben wie:
For more information about the implementation visit our [wiki](link).

**Review by Paula-Kli — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -2,10 +2,23 @@
...
 Metacello new
-	baseline: 'OhmPrettyPrint';
-	repository: 'github://HPI-SWA-Teaching/OhmPrettyPrint:dev/packages';
+	baseline: 'OmegaPrint';
+	repository: 'github://HPI-SWA-Teaching/OmegaPrint:dev/packages';
 	load
 ```
+
+After installing OmegaPrint you will find a button in all your code holder. For example the browser. 
+Clicking on it OmegaPrint will show you a prettified version of your code.
```
Comment (T4rikA) [inline README.md]: When you click on it OmegaPrint will show you ...

File: README.md [hunk 3]
```diff
@@ -2,10 +2,23 @@
...
+
+## Comments
+OmegaPrint does basic comment-formatting.  
+Comment-formatting is more complicated than code-formatting.  
+This happens due to the fact, that comments and whitespace is not in the CST.
```
Comment (T4rikA) [inline README.md]: are not in the CST

File: README.md [hunk 4]
```diff
@@ -2,10 +2,23 @@
...
 ```
+
+After installing OmegaPrint you will find a button in all your code holder. For example the browser. 
+Clicking on it OmegaPrint will show you a prettified version of your code.
+
+## Comments
+OmegaPrint does basic comment-formatting.  
+Comment-formatting is more complicated than code-formatting.  
+This happens due to the fact, that comments and whitespace is not in the CST.  
+Therefore, we use the method ``OhmNode >> skippedSpacesNodes`` which gives us nodes containing all spaces and comments, which are before the node on which we call the method.
```
Comment (T4rikA) [inline README.md]: a node containing all spaces and comments before the node on which we called `skippedSpacesNodes`

**Review by T4rikA — APPROVED** [review]

**Review by felixauringer — APPROVED** [review]

**Pull request merged**